app/services/civil_servant_search.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+class CivilServantSearch
```

Comment:
I think, this name is a bit misleading, since it provides options for a select...